ketabdan/urls.py

```diff
@@ -14,7 +14,7 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path, include
+from django.conf.urls import path, include, url 
 from staticpages.views import IndexView
```

Comment:
When you import something, you should use them too otherwise, there is no point to push your code up without functionality.